web/service/README.md

```diff
@@ -129,3 +129,7 @@ npm run dev
 ```bash
 gcloud builds submit
 ```
+
+## Open API Spec
+
+Open API Spec V3 documentation is avaliable at `web/service/open-api-spec.json`.
```

Comment:
As agreed, add a _"Subject to Change"_ disclaimer in the README and in the swagger file.